Makefile

```diff
@@ -38,9 +51,11 @@ M_INTEG_DIR ?= all
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
 # Image URL to use all building/pushing image targets
-IMG ?= $(IMAGE_TAG_BASE):latest
+IMG ?= $(IMAGE_TAG_BASE):$(VERSION)
+IMG_LATEST ?= $(IMAGE_TAG_BASE):latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+#CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
```

Comment:
Should we leave this line commented out since we're setting it explicitly in the next line?

Reply:
Yes, these are for older versions of controller-gen (only `crd` is what we'll use in newer ones)